Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
```

Comment:
that's fine - you can always modify the 'find' cmd to grab whatever files you want. I was just looking at https://github.com/docker/containerd/blob/master/Makefile#L101 as a guide.